### PR TITLE
Make dupbuild=err work in subninja

### PR DESCRIPTION
**Pull request by sgraham**
Was only an error in top level .ninja previously.

### CODE REVIEW

File: src/manifest_parser_test.cc [hunk 1]
```diff
@@ -24,7 +24,7 @@
 struct ParserTest : public testing::Test,
                     public ManifestParser::FileReader {
   void AssertParse(const char* input) {
-    ManifestParser parser(&state, this);
+    ManifestParser parser(&state, this, false);
```
Comment (nico) [inline src/manifest_parser_test.cc]: nit: s:false:/*dupe_edge_shoud_err=*/false

**Comment by nico** [comment]
Thank you very much for the fix!

File: src/manifest_parser_test.cc (hunk 1 above)
Comment (sgraham) [inline src/manifest_parser_test.cc]: Did you want me to send a followup for all the |false|s? It seemed a bit noisy to put it in every test.

Comment (nico) [inline src/manifest_parser_test.cc]: I probably should've made it an enum parameter, not a bool parameter. If you want, you can either add all the comments or change it to enum; or you can do nothing and I'll do the enum thing eventually.

**Pull request merged**